corehq/apps/export/utils.py

```diff
@@ -165,6 +165,10 @@ def convert_saved_export_to_export_instance(domain, saved_export, dryrun=False):
                     ))
 
                 new_column = _get_normal_column(new_table, column_path, transform)
+
+                # If we can't find the column in the current table
+                # look in every other table to see if the column is a repeat
+                # that did not receive more than one entry
```

Comment:
no it's for the situation when a repeat question doesn't "register" as a repeat question in the old export. if a repeat receives one or less nodes, it doesn't get its own table, thus the new exports doesn't properly match it. does that make sense?